registry/templates/donor/detail.html

```diff
@@ -1,92 +1,156 @@
 {% extends "layout.html" %}
 {% block content %}
 
+
 <h1>Detail dárce</h1>
-<div class="row">
-    <div class="col">
-        <h2>Aktuální informace</h2>
-
-        <ul>
-            <li>Rodné číslo: {{ overview.rodne_cislo }}</li>
-            <li>Jméno: {{ overview.first_name }}</li>
-            <li>Příjmení: {{ overview.last_name }}</li>
-            <li>Adresa: {{ overview.address }}</li>
-            <li>Město: {{ overview.city }}</li>
-            <li>PSČ: {{ overview.postal_code }}</li>
-            <li>Pojišťovna: {{ overview.kod_pojistovny }}</li>
-        </ul>
+<div class="row my-5">
+
+    <div class="col-auto ml-4 mb-4">
+        <h4 class="text-muted border-bottom">Aktuální informace</h4>
+        <div class="container">
+            <div class="row">
+                Rodné číslo: {{ overview.rodne_cislo }}
+            </div>
+            <div class="row">
+                Jméno: {{ overview.first_name }}
+            </div>
+            <div class="row">
+                Příjmení: {{ overview.last_name }}
+            </div>
+            <div class="row">
+                Adresa: {{ overview.address }}
+            </div>
+            <div class="row">
+                Město: {{ overview.city }}
+            </div>
+            <div class="row">
+                PSČ: {{ overview.postal_code }}
+            </div>
+            <div class="row">
+                Pojišťovna: {{ overview.kod_pojistovny }}
+            </div>
+        </div>
     </div>
-    <div class="col">
-        <h2>Počty darování</h2>
 
-        <ul>
+    <div class="col-auto ml-4 mb-4">
+        <h4 class="text-muted border-bottom">Počty darování</h4>
+
+        <div class="container">
             {% for donation_center in donation_centers %}
-                <li>{{ donation_center.title }}: {{ overview["donation_count_" + donation_center.slug] }}</li>
+                <div class="row border-bottom">
+                    {{ donation_center.title | replace(", ", ",<br>" | safe) }}: {{ overview["donation_count_" + donation_center.slug] }}
```

Comment:
@frenzymadness Je tento způsob zalomení pro nás OK

Reply:
Jako hack super :smile: a když je to bez toho zalomení, tak je to špatně proč?

Reply:
Je to strašně dlouhý a nebo se to zalamuje způsobem, že je číslo utopené

Reply:
A nebylo by lepší vložit mezi dvojtečku a číslo nedělitelnou mezeru aby se číslo vždy zalomilo alespoň s jedním slovem?

Reply:
zvolil jsem to timto zpusobem protoze carka oddeluje vetsinou dva logicky celky. Ovsem nemam uplne paru jak ruznorode mohou byt klice pred cislem, tve reseni nejspis bude asi univerzalnejsi pro velkou ruznorodost klici, zkusim na to jeste mrknout a poslu popripade zmenu

Reply:
no trochu je problém s tím že `nbsp` zalamuje pouze když je k tomu donucen a v bootstrapu se mi nedaří nastavit optimální fixní šířku sloupce, aby se nastavila například na šířku nadpisu. V případě, že dám nejmenší možnou šířku `col-sm` šířka sloupce je stále zbytečně široká a má nelogicky velký přesah.

Reply:
nakonec jsem to vyřešil tak, že se tam vleze celý řádek - nejsou tedy nutné už "linky" a design je tak konzistentní pro všechny sloupce. Pro případ že by se někdy vyskytl dlouhý klíč jsem tam vložil `&nbps;`